Support login.json in root app folder on windows

LazeMSS opened the pull request:
Basically allow both the root folder and 64bit folder support when importing the config.json on windows

spuder commented:
Thanks, Line 345 also needs to be updated (both `config import` and `config decode` use the same logic)

LazeMSS commented:
> Thanks, Line 345 also needs to be updated (both `config import` and `config decode` use the same logic)
Fixed and also added the same "logic" for the error messages.

chrivers commented:
LGTM - thanks for your contribution :)

I'll see if I can rework that shared logic, so we can avoid the duplication. But this is good for now :+1:
